udp-native/src/main/java/org/ballerinalang/stdlib/udp/nativelistener/Listener.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.stdlib.udp.nativelistener;
+
+import io.ballerina.runtime.api.Environment;
+import io.ballerina.runtime.api.Future;
+import io.ballerina.runtime.api.utils.StringUtils;
+import io.ballerina.runtime.api.values.BMap;
+import io.ballerina.runtime.api.values.BObject;
+import io.ballerina.runtime.api.values.BString;
+import org.ballerinalang.stdlib.udp.Constants;
+import org.ballerinalang.stdlib.udp.UdpFactory;
+import org.ballerinalang.stdlib.udp.UdpListener;
+import org.ballerinalang.stdlib.udp.UdpService;
+import org.ballerinalang.stdlib.udp.Utils;
+
+import java.net.BindException;
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+
+/**
+ * Native function implementations of the UDP Listener.
+ */
+public class Listener {
+    public static Object initEndpoint(BObject listener, int localPort, BMap<BString, Object> config) {
```

Comment:
Let's just say init instead of initEndpoint

Reply:
changed